### PR TITLE
[FEATURE] Improve I18n section + add crowdin

### PR DESCRIPTION
**Pull request by georgringer**
Improve the Internationalization section + start with docs about crowdin integration

### CODE REVIEW

File: Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/ExtensionIntegration.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+   **Important to know**
+
+   - Your extension must be on GitHub, BitBucket or GitLab
+   - Currently TYPO3 can only handle one branch/version for languages, more to that later.
+
+Setup
+=====
+
+Get in contact with the team in the Team in the TYPO3 slack channel *cig-crowdin-localization* with the following information:
```
Comment (jonaseberledmind) [inline Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/ExtensionIntegration.rst]: with the team in the Team -> with the team

File: Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/ExtensionIntegration.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+Go to the url of your project at Crowdin (e.g. `https://crowdin.com/project/typo3-extension-ttaddress`_) and switch to **Settings** & then switch to the tab **Integrations**. Click on the button **Setup integration**.
+
+.. figure:: ../../../../Images/I18n/Crowdin/crowdin-integration-setup.png
+   :alt: Start of Crowdin integration for an extension
+   :width: 600px
+
+   Start of Crowdin integration for an extension
+
+A modal will open and will allow to to select the proper repository.
```
Comment (jonaseberledmind) [inline Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/ExtensionIntegration.rst]: will allow **you** to select

File: Documentation/ApiOverview/Internationalization/XliffApi.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+convenience wrapper around the :php:`\TYPO3\CMS\Core\Localization\LanguageService` class,
+whose :php:`translate()` method also takes an array as argument and runs PHP's
+:php:`vsprintf()` on the localized string. However, in the future it is expected this Extbase
+specific class will melt down and somehow merged into the core API classes to get rid of this
+duplication.
+
+Label Access in Fluid
+---------------------
+
+In PHP, a typical call to fetch a string in the language selected by a user looks like this:
```
Comment (sypets) [inline Documentation/ApiOverview/Internationalization/XliffApi.rst]: ~PHP~ -> Fluid

**Comment by georgringer** [comment]
@sypets fixed all comments, thx a lot!

**Comment by DanielSiepmann** [comment]
@sypets all changes were applied, and it looked good to me.
As a new TYPO3 version will be released soon, this docs should be available to cover the state.

If there is still something unclear, we should patch it afterwards.

**Comment by sypets** [comment]
Thank you for submitting and for merging. :thumbsup:

**Pull request merged**